secure/storage/vault/src/lib.rs

```diff
@@ -130,7 +140,11 @@ impl Client {
                 Ok(policies.policies)
             }
             // There are no policies.
-            404 => Ok(vec![]),
+            404 => {
+                // Explicitly clear buffer so the stream can be re-used.
```

Comment:
If it were me, I'd define a helper function (e.g., "clear_response_buffer(..)") that contains the comment and the resp.into_string()? call. That way we don't need to duplicate the comment everywhere. 

Of course, the best approach would probably be to wrap ureq somehow to do this automatically for us (that is, assuming we won't get an upstream fix).. You know me and duplication :P

Reply:
Gonna sit on it for now... I would like to see this issue fixed upstream... or to explore creating some sort of droppable wrapper.

Reply:
Thanks for the review!!